PickNikRobotics/lab_sim
# Port over example behaviors from `moveit_studio_example_behaviors`

Partially resolves https://github.com/PickNikRobotics/moveit_studio/issues/8992.

This PR ports over the example behaviors and the example objective that implements the custom MTC behaviors from https://github.com/PickNikRobotics/moveit_studio_example_behaviors/tree/main so that that repo can be deprecated. 

## To test
1) Compile/build this branch
2) Run the `Pick and Place Example` Objective and ensure that the robot picks up and puts down the block.
